execution.bs

```diff
@@ -657,6 +672,10 @@ struct _retry_sender {
   friend _op<S, R> tag_invoke(exec::connect_t, _retry_sender&& self, R r) {
     return {(S&&) self.s_, (R&&) r};
   }
+
+  friend decltype(auto) tag_invoke(exec::get_attrs_t, const _retry_sender& self) noexcept {
```

Comment:
minor nit - the spec for `get_attrs` doesn't say it's always noexcept (unless I missed it), so we need something like `noexcept(noexcept(get_attrs(self.s_))`